x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_define_rule/index.tsx

```diff
@@ -440,14 +406,14 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
       thresholdCardinalityValue,
     }: Record<string, FieldHook>) => (
       <ThresholdInput
-        browserFields={aggFields}
+        browserFields={termsAggregationFields}
```

Comment:
this change can break existing workflows, if customers use date field for aggregation: https://github.com/elastic/kibana/issues/144795